### ISSUE TITLE
I want to draw my architecture, what should I do?

### ISSUE THREAD

**Issue by Viet0904** [HarisIqbal88/PlotNeuralNet#153]
NUM_CLASSES = 8

model = models.Sequential(
    [
        layers.Conv2D(32, (3, 3), input_shape=(224, 224, 3)),
        layers.BatchNormalization(),
        layers.Activation("relu"),
        layers.MaxPooling2D((2, 2)),
        layers.Conv2D(64, (3, 3)),
        layers.BatchNormalization(),
        layers.Activation("relu"),
        layers.MaxPooling2D((2, 2)),
        layers.Conv2D(128, (3, 3)),
        layers.BatchNormalization(),
        layers.Activation("relu"),
        layers.MaxPooling2D((2, 2)),
        layers.Flatten(),
        layers.Dense(512),
        layers.BatchNormalization(),
        layers.Activation("relu"),
        layers.Dropout(0.5),
        layers.Dense(NUM_CLASSES, activation="softmax"),
    ]
)
Above is my architecture. Tôi đã thay thử 
# defined your arch
arch = [
    to_head(".."),
    to_cor(),
    to_begin(),
    to_Conv(
        "conv1", 224, 32, offset="(0,0,0)", to="(0,0,0)", height=40, depth=40, width=2
    ),
    to_Pool("pool1", offset="(0,0,0)", to="(conv1-east)"),
    to_Conv(
        "conv2",
        196,
        64,
        offset="(2,0,0)",
        to="(pool1-east)",
        height=40,
        depth=40,
        width=2,
    ),
    to_Pool("pool2", offset="(0,0,0)", to="(conv2-east)", height=36, depth=36, width=1),
    to_Conv(
        "conv3",
        168,
        128,
        offset="(2,0,0)",
        to="(pool2-east)",
        height=36,
        depth=36,
        width=2,
    ),
    to_Pool("pool3", offset="(0,0,0)", to="(conv3-east)", height=32, depth=32, width=1),
    to_Conv(
        "flatten",
        16,
        1,
        offset="(2,0,0)",
        to="(pool3-east)",
        height=1,
        depth=1,
        width=2,
    ),
    to_Dense(
        "dense1",
        512,
        offset="(2,0,0)",
        to="(flatten-east)",
        height=64,
        depth=64,
        width=2,
    ),
    to_Dropout("dropout1", "(1,0,0)", "(dense1-east)", caption="Dropout"),
    to_SoftMax("softmax", 8, "(1,0,0)", "(dropout1-east)", caption="Softmax"),
    to_end(),
]
and it doesn't seem right. Someone please help me
